react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.state = {
+      "one": "hidden",
+      "two": "hidden",
+      "three": "hidden"
+    }
+    this.handleOpen = this.handleOpen.bind(this);
+    this.handleClose = this.handleClose.bind(this);
+  }
+
+  render(props) {
+    let prop = this.props.content;
+    let state = this.state;
+
+    return (
+      <div className="container" onDoubleClick={this.handleClose}>
+        <div className="one" onClick={this.handleOpen}>
+          {prop[0].name}
+        </div>
+        <div className={`text one ${state.one}`}>{prop[0].description}</div>
+        <div className="two" onClick={this.handleOpen}>
+          {prop[1].name}
+        </div>
+        <div className={`text two ${state.two}`}>{prop[1].description}</div>
+        <div className="three" onClick={this.handleOpen}>
+          {prop[2].name}
+        </div>
+        <div className={`text three ${state.three}`}>{prop[2].description}</div>
+      </div>
```

Comment:
Here you should be using a loop to generate your JSX so no matter how many items are passed in your accordion will still work and display all items. It would look something like:

```JSX
<div className="container" onDoubleClick={this.handleClose}>
  {
    prop.map(item => (
      <div key={item.id}>
        <div className="accordion-item" onClick={() => this.handleOpen(item.id)}>
          {item.name}
        </div>
        <div className={`accordion-description ${state.openId === item.id ? 'on' : 'hidden'}`}>{item.description}</div>
      </div>
    ))
  }
</div>
```